public/css/module.less

```diff
@@ -255,6 +252,8 @@ ul.bp {
         }
 
         .overridden-state {
+            display: flex;
+            align-items: center;
```

Comment:
On master: ![bp-override-on-master](https://github.com/Icinga/icingaweb2-module-businessprocess/assets/16668527/222111c2-2a7e-4756-b51c-3b7db47c3085)
Here: ![bp-override-with-new-icon](https://github.com/Icinga/icingaweb2-module-businessprocess/assets/16668527/090422f3-438a-4493-aaab-88cf2c6767ec)

Neither looks fine to me.